### PR TITLE
Qm fixes

### PR DESCRIPTION
**Pull request by jordivallsq**
Removed the timeout error check (as it has been fixed) and fixed shareable

### CODE REVIEW

File: src/qililab/qprogram/quantum_machines_compiler.py [hunk 1]
```diff
@@ -117,7 +117,6 @@ class QuantumMachinesCompiler:
 
     FREQUENCY_COEFF = 1
     PHASE_COEFF = 2 * np.pi
-    VOLTAGE_COEFF = 2
```
Comment (fedonman) [inline src/qililab/qprogram/quantum_machines_compiler.py:120]: Why did we remove the VOLTAGE_COEFF? Did their ranges change?

**Review by fedonman — COMMENTED** [review]
Aside from the question on voltages, rest looks good! 💯

File: src/qililab/qprogram/quantum_machines_compiler.py (hunk 1 above)
Comment (jordivallsq) [reply src/qililab/qprogram/quantum_machines_compiler.py:120]: Yes, QM changed the way they use their amplitudes (both in waveforms and in the gains inside a play/measure). I tested this in an oscilloscope, I can send you the details through slack (images and the config file).

Comment (fedonman) [reply src/qililab/qprogram/quantum_machines_compiler.py:120]: Yes please send the details. Also we should document this in the changelog.

File: src/qililab/qprogram/quantum_machines_compiler.py [hunk 2]
```diff
@@ -617,10 +616,10 @@ def __hash_waveform(waveform: Waveform):
     @staticmethod
     def __waveform_to_config(waveform: Waveform):
         if isinstance(waveform, Square):
-            amplitude = waveform.amplitude / QuantumMachinesCompiler.VOLTAGE_COEFF
+            amplitude = waveform.amplitude
             return {"type": "constant", "sample": amplitude}
 
-        envelope = waveform.envelope() / QuantumMachinesCompiler.VOLTAGE_COEFF
+        envelope = waveform.envelope()
```
Comment (fedonman) [inline src/qililab/qprogram/quantum_machines_compiler.py:622]: here too

File: src/qililab/platform/platform.py [hunk 3]
```diff
@@ -913,48 +907,36 @@ def _execute_quantum_machines_compilation_output(
...
-                    )
-                    measurement_result.set_classification_threshold(measurement.threshold)
-                    results.append_result(bus=measurement.bus, result=measurement_result)
-
-                return results
-
-            except TimeoutError as timeout:
-                warnings.warn(
-                    f"Warning: {timeout} raised, retrying experiment ({iteration + 1}/{timeout_tries} available tries)"
+        try:
```
Comment (fedonman) [inline src/qililab/platform/platform.py:913]: Add a changelog entry for removing the temporal fix.

**Review by fedonman — CHANGES_REQUESTED** [review]

File: src/qililab/qprogram/qblox_compiler.py [hunk 4]
```diff
@@ -162,6 +162,7 @@ def compile(
         times_of_flight: dict[str, int] | None = None,
         delays: dict[str, int] | None = None,
         markers: dict[str, str] | None = None,
+        voltage_coefficient: dict[str, float] | None = None,
```
[the remark this answers is not in the document]
Comment (jordivallsq) [reply src/qililab/qprogram/qblox_compiler.py]: removed

File: src/qililab/qprogram/quantum_machines_compiler.py [hunk 5]
```diff
@@ -395,7 +394,7 @@ def _handle_play(self, element: Play):
         duration = waveform_I.get_duration()
 
         gain = (
-            qua.amp(self._buses[element.bus].current_gain * self.VOLTAGE_COEFF)  # type: ignore[arg-type]
+            qua.amp(self._buses[element.bus].current_gain)  # type: ignore[arg-type]
```
[the remark this answers is not in the document]
Comment (jordivallsq) [reply src/qililab/qprogram/quantum_machines_compiler.py]: added

File: src/qililab/platform/platform.py (hunk 3 above)
Comment (jordivallsq) [reply src/qililab/platform/platform.py:913]: Added the text in the changelog

**Comment by jordivallsq** [comment]
I removed any change in the Voltage coefficient and I will create a task focused on it

**Review by fedonman — APPROVED** [review]
great work :100:

**Pull request merged**